docs/guided-tour/blueprints/helm-chart-resource/README.md

```diff
@@ -85,8 +90,7 @@ components:
           helmRepository: https://example.helm.repo.com/landscaper
 ```
 
-
-Again, if you were to prefer to embed the blueprint and the helm chart in the component as a local blob, instead of an 
+Again, if you prefer to embed the blueprint and the helm chart in the component as a local blob, instead of an 
```

Comment:
This is not correct. A resource being a local blob is not determined if it is specified by `access` or `input`.
You can use `input` and not have a local blob.

Reply:
We explain this during the rework of the chapter. We should probably link to https://ocm.software/docs/guides/input_and_access/